src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -360,7 +351,7 @@ public override bool IsVararg
             get
             {
                 LazyMethodChecks();
-                return _lazyIsVararg;
+                return flags.IsVarArg;
```

Comment:
>return flags.IsVarArg; [](http://example.com/codeflow?start=16&length=22)

Do we still need to do ```LazyMethodChecks``` before this value is valid? If not, the property should be moved to the base that declares ```flags``` and be "sealed" there.

Reply:
sure.